client/src/utils.rs

```diff
@@ -60,24 +62,28 @@ pub fn get_councillors(api: &Api) -> Result<Vec<AccountId>> {
 		.ok_or_else(|| ApiClientError::Other("Couldn't get councillors".into()))
 }
 
-pub fn send_and_wait_for_in_block<C: Encode>(
+pub fn send_and_wait_for_in_block<C: Encode + Clone>(
 	api: &Api,
 	xt: EncointerXt<C>,
 	tx_payment_cid: Option<&str>,
 ) -> Option<H256> {
-	send_xt_hex_and_wait_for_in_block(api, xt.hex_encode(), tx_payment_cid)
+	send_xt_hex_and_wait_for_in_block(api, xt, tx_payment_cid)
 }
 
-pub fn send_xt_hex_and_wait_for_in_block(
+pub fn send_xt_hex_and_wait_for_in_block<C>(
 	api: &Api,
-	xt_hex: String,
+	xt: EncointerXt<C>,
 	tx_payment_cid: Option<&str>,
-) -> Option<H256> {
-	ensure_payment(api, &xt_hex, tx_payment_cid);
-	let tx_hash = api.send_extrinsic(xt_hex, XtStatus::InBlock).unwrap();
+) -> Option<H256>
+where
+	C: Clone + Encode,
+{
+	let encoded_xt = hex::encode(xt.encode());
```

Comment:
To encode an extrinsic into `Bytes`: use `xt.encode().into()` No need to use strings anymore. Or if you want to avoid `Bytes`: Use `xt.encode()` and a `Vec<u8>` and when using  the api-client function, simply use `encoded_xt.into()`.

(check this out: https://github.com/scs/substrate-api-client/blob/master/testing/examples/pallet_transaction_payment_tests.rs#L36-L43)